### PR TITLE
[OTX][Unit Test] Tiling Detection UnitTest

### PR DESCRIPTION
**Pull request by eugene123tw**
* Add tiling detection test

Screenshot from 2023-01-31 14-49-43

* Fix OpenVINO tiling when config.json is missing

### CODE REVIEW

File: tests/unit/algorithms/detection/tiling/test_tiling_detection_unittest.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    image = Image(data=image)
+    annotation_scene = AnnotationSceneEntity(annotations=anno_list, kind=AnnotationSceneKind.ANNOTATION)
+    dataset_item = DatasetItemEntity(media=image, annotation_scene=annotation_scene)
+    return DatasetEntity([dataset_item]), labels
+
+
+class TilingDetectionUnitTest(unittest.TestCase):
+    """Test the tiling functionality"""
+
+    def setUp(self) -> None:
```
Comment (jaegukhyun) [inline tests/unit/algorithms/detection/tiling/test_tiling_detection_unittest.py:45]: https://github.com/openvinotoolkit/training_extensions/blob/f75cf367b9aa8caeabcc9134a9493e77e7d3c7cb/tests/unit/cli/builder/test_cli_builder.py#L50
How about sync with existing test?

**Comment by sungmanc** [comment]
This PR was using the `unittest`, not using `pytest`. Most of other unit tests are using `pytest` for implementation.
So, I suggest changing the library. 

What's your opinion? @samet-akcay , @goodsong81 , @sovrasov , @harimkang

**Comment by harimkang** [comment]
> 

I think this can be done later? In fact, there doesn't seem to be a big problem with test itself (Because unittest can be collected into pytest.), and we don't have much time to write test code...

**Comment by goodsong81** [comment]
No problem. We agreed up on that `class TestA` could be either of simple class or inheritance of `Unittest.TestCase`.
BTW, `TestA` is our consensus. @eugene123tw Could you change the name?
(We thought `Test` prefix is a must-have... but it seems not? :p)

**Comment by eugene123tw** [comment]
> No problem. We agreed up on that `class TestA` could be either of simple class or inheritance of `Unittest.TestCase`. BTW, `TestA` is our consensus. @eugene123tw Could you change the name? (We thought `Test` prefix is a must-have... but it seems not? :p)

Done! Thanks!

**Review by goodsong81 — DISMISSED** [review]
Thank you :)

**Comment by eugene123tw** [comment]
```
2023-02-02T11:31:58.2005725Z =========================== short test summary info ============================
2023-02-02T11:31:58.2006607Z SKIPPED [1] tests/integration/cli/classification/test_classification.py:158: CVS-101246 Multi-GPU tests are stuck while CI is running
2023-02-02T11:31:58.2007672Z SKIPPED [1] tests/integration/cli/detection/test_detection.py:140: CVS-101246 Multi-GPU tests are stuck while CI is running
2023-02-02T11:31:58.2008714Z SKIPPED [1] tests/integration/cli/detection/test_instance_segmentation.py:124: CVS-101246 Multi-GPU tests are stuck while CI is running
2023-02-02T11:31:58.2009737Z SKIPPED [1] tests/integration/cli/detection/test_tiling_detection.py:97: CVS-98026
2023-02-02T11:31:58.2010562Z SKIPPED [1] tests/integration/cli/detection/test_tiling_instseg.py:97: CVS-98026
2023-02-02T11:31:58.2011524Z SKIPPED [1] tests/integration/cli/segmentation/test_segmentation.py:147: CVS-101246 Multi-GPU tests are stuck while CI is running
2023-02-02T11:31:58.2012553Z XFAIL tests/unit/api/usecases/exportable_code/test_streamer.py::TestStreamer::test_threaded_streamer - CVS-102619
2023-02-02T11:31:58.2013423Z XPASS tests/unit/api/entities/test_dataset_item.py::TestDatasetItemEntity::test_dataset_item_get_annotations 
2023-02-02T11:31:58.2014504Z FAILED tests/integration/cli/detection/test_instance_segmentation.py::TestInstanceSegmentationCLI::test_nncf_optimize[Custom_Counting_Instance_Segmentation_MaskRCNN_EfficientNetB2B]
2023-02-02T11:31:58.2015451Z = 1 failed, 834 passed, 6 skipped, 1 xfailed, 1 xpassed, 36 warnings in 5149.95s (1:25:49) =
```
Failed case NNCF optmization. 

```
2023-02-02T11:14:30.1576642Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/model_creation.py", line 102, in create_compressed_model
2023-02-02T11:14:30.1578273Z     compressed_model = builder.apply_to(nncf_network)
2023-02-02T11:14:30.1580518Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/compression_method_api.py", line 122, in apply_to
2023-02-02T11:14:30.1582149Z     transformation_layout = self.get_transformation_layout(model)
2023-02-02T11:14:30.1584308Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/compression_method_api.py", line 139, in get_transformation_layout
2023-02-02T11:14:30.1585931Z     layout = self._get_transformation_layout(model)
2023-02-02T11:14:30.1588133Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/quantization/algo.py", line 620, in _get_transformation_layout
2023-02-02T11:14:30.1589858Z     self._pt_quantizer_setup = self._get_quantizer_setup(target_model)
2023-02-02T11:14:30.1591973Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/quantization/algo.py", line 712, in _get_quantizer_setup
2023-02-02T11:14:30.1593685Z     stats_for_range_init = self._get_statistics_for_final_range_init(target_model,
2023-02-02T11:14:30.1595933Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/quantization/algo.py", line 688, in _get_statistics_for_final_range_init
2023-02-02T11:14:30.1597825Z     return self.get_statistics_for_quantizer_setup(target_model, quantizer_setup, range_init_params)
2023-02-02T11:14:30.1600153Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/quantization/algo.py", line 680, in get_statistics_for_quantizer_setup
2023-02-02T11:14:30.1602101Z     retval[ip] = {rs: collector.get_statistics() for rs, collector in rs_vs_collector.items()}
2023-02-02T11:14:30.1604252Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/torch/quantization/algo.py", line 680, in <dictcomp>
2023-02-02T11:14:30.1605970Z     retval[ip] = {rs: collector.get_statistics() for rs, collector in rs_vs_collector.items()}
2023-02-02T11:14:30.1608171Z   File "/home/validation/actions-runner/_work/training_extensions/training_extensions/.tox/pre-merge/lib/python3.8/site-packages/nncf/common/tensor_statistics/collectors.py", line 66, in get_statistics
2023-02-02T11:14:30.1609736Z     raise StatisticsNotCollectedError()
2023-02-02T11:14:30.1610857Z nncf.common.tensor_statistics.collectors.StatisticsNotCollectedError
2023-02-02T11:14:30.2279689Z FAILED
2023-02-02T11:14:30.2302916Z tests/integration/cli/detection/test_instance_segmentation.py::TestInstanceSegmentationCLI::test_otx_multi_gpu_train[Custom_Counting_Instance_Segmentation_MaskRCNN_ResNet50] SKIPPED
2023-02-02T11:14:40.6306571Z tests/integration/cli/detection/test_tiling_detection.py::TestTilingDetectionCLI::test_otx_train[Custom_Object_Detection_Gen3_ATSS] 2023-02-02 11:14:40,414 | INFO : created output path at /tmp/tmp5s59eroc/OTX-
```

File: tests/unit/algorithms/detection/tiling/test_tiling_detection_unittest.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                    otx_dataset=self.otx_dataset.with_empty_annotations(),
+                    labels=list(self.labels),
+                    domain=Domain.DETECTION,
+                ),
+                test_mode=True,
+                **self.tile_cfg
+            )
+        )
+
+    def test_tiling_train_dataloader(self):
```
Comment (harimkang) [inline tests/unit/algorithms/detection/tiling/test_tiling_detection_unittest.py:120]: If it's a unit test, could you add a unit test decorator `@e2e_pytest_unit`?
`from tests.test_suite.e2e_test_system import e2e_pytest_unit`

**Review by goodsong81 — APPROVED** [review]
LGTM

**Review by jaegukhyun — APPROVED** [review]

**Pull request merged**